doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst

```diff
@@ -95,7 +95,8 @@ LTE modem
 =========
 
 The LTE modem handles LTE communication.
-It is controlled through `AT commands <AT Commands Reference Guide_>`_.
+It is controlled through AT commands.
+The AT commands are documented in the `nRF9160 AT Commands Reference Guide`_.
```

Comment:
This document is not for 91x1?

Reply:
Presently, we just have information about nRF9160 in the working with guides. The Working With guides are yet to be updated with the nRF9161 details.  We have a open task for this. The information about AT commands reference guide would be added then.